airflow/cli/commands/variable.py

```diff
@@ -0,0 +1,174 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Variable subcommands"""
+import json
+import os
+from json import JSONDecodeError
+
+import rich_click as click
+
+from airflow.cli.simple_table import AirflowConsole
+from airflow.models import Variable
+from airflow.utils import cli as cli_utils
+from airflow.utils.cli import suppress_logs_and_warning
+from airflow.utils.session import create_session
+from airflow.cli import airflow_cmd, click_dry_run, click_verbose, click_yes, click_output
+
+click_json = click.option(
+    '-j',
+    '--json',
+    help="Deserialize JSON variable"
+)
+
+click_default = click.option(
+    '-d',
+    '--default',
+    help='Default value returned if variable does not exist'
+)
+
+click_key = click.argument(
+    'key',
+)
+
+click_value = click.argument(
+    'value',
+)
+
+click_variable_file = click.argument(
+    'files',
+    type=click.Path(exists=True),
+)
+
+@airflow_cmd.group()
+@click.pass_context
+def variable(ctx):
+    pass
+
+@suppress_logs_and_warning
```

Comment:
Hi @Bowrna 

How do you think to handle `suppress_logs_and_warning` ? It expects a `verbose` flag as the first function argument.

Reply:
I'm thinking about doing something like this:

```py
@airflow_cmd.command("foo")
@click_verbose
@suppress_logs_and_warning_click_compatible
def cheat_sheet(verbose):
    pass
```
```py
def suppress_logs_and_warning_click_compatible(f: T) -> T:
    @functools.wraps(f)
    def _wrapper(*args, **kwargs):
        if kwargs.get("verbose"):
            f(*args, **kwargs)
        else:
            ... do the same as `suppress_logs_and_warning`
```

Reply:
ah thanks for pointing me to that part @hankehly. I will check yours for more details